### PR TITLE
Add esau-morais to LinkFree

### PR DESCRIPTION
**Pull request by esau-morais**
This PR adds me as part of the GitHub UI

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2792"></a>

### CODE REVIEW

File: data/esau-morais.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Esaú Morais",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "Full-Stack Web Developer",
+  "avatar": "https://avatars.githubusercontent.com/u/55207584?v=4",
```
Comment (Panquesito7) [inline data/esau-morais.json:6]: It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/esau-morais.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

File: data/esau-morais.json (hunk 1 above)
Comment (eddiejaoude) [reply data/esau-morais.json]: This is more dynamic and will use their latest profile picture from GitHub but I don't think it is a blocker

Comment (eddiejaoude) [reply data/esau-morais.json]: Oh it was updated, great 👍

**Pull request merged**